src/service/paranet-id-service.js

```diff
@@ -0,0 +1,14 @@
+import UalService from './ual-service.js';
+
+class ParanetIdService extends UalService {
```

Comment:
This shouldn't be an inheritance, paranet-id-service should use, not extend, UalService, and wherever in code we are use paranet id service as UAL Service it should be switched to Ual Service